shutter-service-1000/variables.env

```diff
@@ -0,0 +1,19 @@
+# Enter variables that should be included in the assets container here, one per line, (ba)sh syntax.
+# convention is to start the names with `_ASSETS_` to prevent accidental naming conflicts
+
+_ASSETS_DISCOVERY_NAME_PREFIX="shutter-service"
+
+_ASSETS_INSTANCE_ID=1000
+_ASSETS_HTTP_ENABLED=false
+_ASSETS_MAX_NUM_KEYS_PER_MESSAGE=500
+_ASSETS_SYNC_START_BLOCK_NUMBER=12128900
+_ASSETS_KEYPER_SET_MANAGER=0x0ACA660D3AaBFb2C6dB6d983BAe4239e8A84Fd02
+_ASSETS_KEY_BROADCAST_CONTRACT=0xdC5E2A1a3e54a3c35051F2EEa021597413672fD0
+_ASSETS_SHUTTERREGISTRY=0x43771dc84F990c062dCc8b308C58bc29a60fDf96
+_ASSETS_CUSTOM_BOOTSTRAP_ADDRESSES='["/ip4/127.0.0.1/tcp/2001/p2p/12D3KooWMyutShWdqYj7fre4Vjuq2QnCTb26Ki1KpyDyVsrmKeki"]'
+_ASSETS_DKG_PHASE_LENGTH=8
+_ASSETS_DKG_START_BLOCK_DELTA=5
```

Comment:
Those are pretty short, on gnosis an 8 block phase is just 40s.

For a test deployment it might be fine but we definitely need to put realistic values here for the actual deployment.